plasmoid/contents/ui/screens.js

```diff
@@ -0,0 +1,49 @@
+function saveConfig() {
+    console.log("Saving config")
+    cfg_screenNameList = [];
+    cfg_screenColorList = [];
+
+    for (var i = 0; i < screenListModel.count; i++) {
+        var name = screenListModel.get(i).screenName;
+        var color = Qt.lighter(screenListModel.get(i).screenColor, 1);
```

Comment:
Does `Qt.lighter` with a factor of 1 actually do anything?

Reply:
This is a workaround. I couldn't find a method to create `QColor` from `string` in Javascript, so I used `Qt.lighter()` with the factor of `1` to do this. Is there a normal way to do this?

Reply:
Huh, that's weird. The closest thing I can find is Qt.rgba, but that function doesn't offer the same api. 
Maybe create a helper function with a short comment about why Qt.lighter is used? Or a helper function that uses Qt.rgba.